### PR TITLE
Add Kotlinter to check Kotlin code style

### PR DESCRIPTION
**Pull request by Goooler**
## Description

https://github.com/jeremymailen/kotlinter-gradle

```editorconfig
[*.{kt,kts}]
ij_kotlin_imports_layout = *
ij_kotlin_allow_trailing_comma = true
ij_kotlin_allow_trailing_comma_on_call_site = true
```

## Type of change

:x: Bug fix (non-breaking change which fixes an issue)
:x: Icon addition (non-breaking change that adds/modifies Lawnicons's icons)
:white_check_mark: General change (non-breaking change that doesn't fit the above categories like copyediting)

### CODE REVIEW

File: .github/workflows/check.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+name: Build debug APK
```
Comment (SuperDragonXD) [inline .github/workflows/check.yml:1]: Shouldn't this be changed to "Prettify Kotlin Code", "Check Kotlin Style" or something similar?

Also I think having a more descriptive file name (like `check_kotlin_style`) could be better too.

**Review by SuperDragonXD — CHANGES_REQUESTED** [review]
Just one small suggestion on the `check.yml` file.

File: .github/workflows/check.yml (hunk 1 above)
Comment (Goooler) [reply .github/workflows/check.yml]: Ohhhh, I copied from `build_debug_apk.yml`, missed it.

Comment (Goooler) [reply .github/workflows/check.yml]: f782a64c84270ccdf94525aed0aa23bfb381c2ce

**Review by SuperDragonXD — APPROVED** [review]

File: .github/workflows/check_kotlin_style.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  push:
+    branches:
+      - develop
+    path:
+      - '.**/**.kt'
+      - '.**/**.kts'
+  pull_request:
+    path:
+      - '.**/**.kt'
+      - '.**/**.kts'
```
Comment (Goooler) [inline .github/workflows/check_kotlin_style.yml:13]: Path filter not work, fixed by bc3966190d27a3310d1d1b65f8e0df060d799fd2

**Pull request merged**